### ISSUE TITLE
Train data like all words,patterns are not well handled. Patterns ,all words are not appended

### ISSUE THREAD

**Issue by Sawant1007** [chicken-burgerrrr/medico_medical_chatbot#3]
X_train is not append with the  bag of words for each pattern_sentence.
Y_train:  PyTorch CrossEntropyLoss needs only class labels, not one-hot is also not maintained
